src/main/java/de/teamlapen/vampirism/world/gen/biome/VampireBiome.java

```diff
@@ -0,0 +1,18 @@
+package de.teamlapen.vampirism.world.gen.biome;
+
+import de.teamlapen.vampirism.api.VReference;
+import de.teamlapen.vampirism.api.VampirismAPI;
+import de.teamlapen.vampirism.api.entity.factions.IFaction;
+import de.teamlapen.vampirism.api.world.IFactionBiome;
+
+public abstract class VampireBiome extends VampirismBiome implements IFactionBiome {
```

Comment:
Not really necessary to add another class that has only one subclass (for the forseable future) and does very little, but ok